pom.xml

```diff
@@ -46,8 +46,67 @@
+                                <includes>
+                                    <include>*.md</include>
+                                    <include>.gitignore</include>
+                                    <include>.java</include>
+                                </includes>
+                                <!-- define the steps to apply to those files -->
+                                <trimTrailingWhitespace/>
+                                <endWithNewline/>
+                                <indent>
+                                    <tabs>true</tabs>
+                                    <spacesPerTab>4</spacesPerTab>
+                                </indent>
+                            </format>
+                        </formats>
+                        <!-- define a language-specific format -->
+                        <java>
+                            <includes>
+                                <include>src/main/java/**/*.java</include>
+                                <include>src/test/java/**/*.java</include>
+                            </includes>
+                            <importOrder/>
+                            <removeUnusedImports/>
+                            <!-- no need to specify files, inferred automatically, but you can if you want -->
+                            <!-- apply a specific flavor of google-java-format and reflow long strings -->
+                            <googleJavaFormat>
+                                <version>1.8</version>
+                                <style>AOSP</style>
+                                <reflowLongStrings>true</reflowLongStrings>
+                            </googleJavaFormat>
+                            <!-- make sure every file has the following copyright header.
+                              optionally, Spotless can set copyright years by digging
+                              through git history (see "license" section below) -->
+                            <!--                            <licenseHeader>-->
+                            <!--                                <content>/* (C)$YEAR */-->
+                            <!--                                </content>  &lt;!&ndash; or <file>${project.basedir}/license-header</file> &ndash;&gt;-->
+                            <!--                            </licenseHeader>-->
+                        </java>
+                    </configuration>
+                    <executions>
+                        <execution>
+                            <!-- Runs in compile phase to fail fast in case of formatting issues.-->
+                            <id>spotless-apply</id>
+                            <phase>compile</phase>
+                            <goals>
+                                <goal>apply</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
```

Comment:
Move all the configuration to <plugins></plugins> section outside of pluginManagement
```xml
 <plugins>
            <plugin>
                <groupId>com.diffplug.spotless</groupId>
                <artifactId>spotless-maven-plugin</artifactId>
                <configuration>
                ...
            </plugin>
 </plugins>
```

In pluginManagement section left only plugin declaration
```xml
  <pluginManagement>
            <plugins>
                <plugin>
                    <groupId>com.diffplug.spotless</groupId>
                    <artifactId>spotless-maven-plugin</artifactId>
                    <version>2.24.1</version>
                </plugin>
            </plugins>
        </pluginManagement>
```